src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import socialMedia from '../../assets/socialMedia.svg';
+import logo from '../../assets/logo.svg';
+
+const Footer: React.FC = () => {
```

Comment:
We will not use `React.FC` method. It's a primitive way of defining typeScript which are not being followed by TypeScript community now a days. You can remove it